src/components/MenuItem/MenuItem.stories.tsx

```diff
@@ -2,9 +2,17 @@
 
 import IconAudio from "@foundation/Icon/Generated/IconAudio";
 import { Meta, Story } from "@storybook/react";
-import React from "react";
+import React, { useState } from "react";
 import { MenuItem as MenuItemComponent, MenuItemProps, MenuItemStyle, SelectionIndicatorIcon } from "./MenuItem";
 import { MenuItemContentSize } from "./MenuItemContent";
+import { Switch, SwitchSize } from "@components/Switch";
+
+const SwitchComponent = () => {
+    const [switchValue, setSwitchValue] = useState<boolean>(false);
+    const toggleSwitch = () => setSwitchValue(!switchValue);
+
+    return <Switch size={SwitchSize.Small} on={switchValue} onChange={toggleSwitch} />;
```

Comment:
I guess the size could be configurable, but since I didn't see any other design and we're trying to limit flexibility (if I understood that right), I just used a hardcoded value. But we can always add another prop to define the Switch size. To the second point, the `Switch` needs to be defined in `AriaMenuItem` in order to be able to react to the Menu Item being clicked (i.e. same place where the `onClick` is defined). Having the `Switch` within the Menu Item would mean I have to pass additional props, to propagate the Switch value up the component tree and be able to toggle it.